README.md

```diff
@@ -3,40 +3,114 @@
+
+- [Governance dashboard](https://github.com/open-cluster-management/grc-ui) -- UI
+- Govenance policy framework -- A framework to distribute various supported policies to managed cluster and collect results back to hub. It consists of following components:
+    - [Policy propagator](https://github.com/open-cluster-management/governance-policy-propagator) 
+    - [Policy spec sync](https://github.com/open-cluster-management/governance-policy-spec-sync)
+    - [Policy status sync](https://github.com/open-cluster-management/governance-policy-status-sync)
+    - [Policy template sync](https://github.com/open-cluster-management/governance-policy-template-sync)
+- Policy controllers -- Policy engines running on managed cluster to evaluate policy rules distributed by the policy framework and generate results.
+    - [Configuration policy controller](https://github.com/open-cluster-management/config-policy-controller)
+    - [Certificate policy controller](https://github.com/open-cluster-management/cert-policy-controller)
+    - [IAM policy controller](https://github.com/open-cluster-management/iam-policy-controller)
+    - 3rd party
+      - [Gatekeeper](https://github.com/open-policy-agent/gatekeeper)
+      - [Kyverno](https://github.com/kyverno/kyverno/)
+      - [Bring your own](#bring-your-own-policy-controller)
+
+## The Policy CRDs
+
+The `Policy` is the Custom Resource Definition (CRD), created for policy framework controllers to monitor. It acts as a vechical to deliver policies to managed cluster and collect results back to hub.
+This is an example spec of a `Policy` object:
+```yaml
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-pod
+spec:
+  remediationAction: inform         # [inform/enforce] If set, it defines the remediationAction globally.
+  disabled: false                   # [true/false] If true, the policy will not be distributed to the managed cluster.
+  policy-templates:             
+    - objectDefinition:             # Use `objectDefinition` to wrap the policy resource to be distributed to the managed cluster
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: policy-pod-example
+        spec:
+          remediationAction: inform
+          object-templates:
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: v1
+                kind: Pod 
+                metadata:
+                  name: sample-nginx-pod
+                  namespace: default
+                spec:
+                  containers:
+                  - image: nginx:1.7.9
+                    name: nginx
+                    ports:
+                    - containerPort: 80
+```
+
+The `PlacementBinding` CRD is used to bind the `Policy` with `PlacementRule`. Only bound `Policy` will be distributed by policy framework to the managed cluster
```

Comment:
Change: The `PlacementBinding` CRD is used to bind the `Policy` with `PlacementRule`.

TO 

The `PlacementBinding` CRD is used to bind the `Policy` with a `PlacementRule`.

Reply:
Only a bound `Policy` is distributed to a managed cluster by the policy framework.